django/db/models/fields/__init__.py

```diff
@@ -828,18 +828,17 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
+    @warn_about_renamed_method('Field', '_get_val_from_obj', 'value_from_object',
+                               RemovedInDjango21Warning)
     def _get_val_from_obj(self, obj):
-        if obj is not None:
```

Comment:
Actually, `get_default` does nothing as defaults are already provided by `model.__init__`. So the implementations are the same.